sink/worldcat/src/main/java/dk/dbc/dataio/sink/worldcat/ChunkItemWithWorldCatAttributes.java

```diff
@@ -31,7 +33,7 @@ public static ChunkItemWithWorldCatAttributes of(ChunkItem chunkItem) throws Ill
                         .withEncoding(chunkItem.getEncoding())
                         .withStatus(chunkItem.getStatus())
                         .withTrackingId(chunkItem.getTrackingId())
-                        .withType(chunkItem.getType().toArray(new ChunkItem.Type[chunkItem.getType().size()]))
+                        .withType(chunkItem.getType().toArray(new Type[0]))
                         .withData(addiRecord.getContentData());
```

Comment:
Hmm det er sgu en lidt sær konstruktion. Både før og efter.

Reply:
Det var mest for at komme af med noget af det gule i IntelliJ